app/views/shared/_flash_nessages.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if flash.any? %>
+    <% flash.each do |msg_type msg| %>
+        <div class="container p-2">
+            <div class="alert alert-<%= msg_type %>">
+                <%= msg %>
+            </div>
+        </div>
+    <% end %>
+<% end %>
```

Comment:
You have incorrect spelling in your file name. This is should be `_flash_messages.html.erb`